geometry/optimization/iris.cc

```diff
@@ -522,6 +530,495 @@ std::vector<int> revolute_joint_indices(const multibody::Joint<double>& joint) {
 
 }  // namespace
 
+namespace {
+
+int FindCollisionPairIndex(const MultibodyPlant<double>& plant, 
```

Comment:
Once we can get the collision pair out of SceneGraphCollisionChecker, we can delete this entire function